src/Libraries/CoreNodeModels/CurveMapperNodeModel.cs

```diff
@@ -678,9 +682,16 @@ public void UpdateGaussianCurveControlPoints(double deltaX, string tag)
 
         private bool IsValidInput()
         {
-            return PointsCount >= 2
-                && MinLimitX != MaxLimitX
-                && MinLimitY != MaxLimitY;
+            if (pointsCount == null || pointsCount.Count == 0)
+                return false;
+
+            if (pointsCount.Count == 1 && pointsCount[0] < 2)
```

Comment:
Curious as to why the first value in the count list should be greater than or equal to 2, for it to be valid? 
Would suggest using firstordefault() instead of indexing the value.